app/controllers/meetings_controller.rb

```diff
@@ -45,72 +26,52 @@ def comment
 
   # rubocop:disable MethodLength
   def delete_comment
-    comment_exists = Comment.where(id: params[:commentid]).exists?
-    is_my_comment = Comment.where(
-      id: params[:commentid],
-      comment_by: current_user.id
-    ).exists?
+    comment = Comment.find_by(id: params[:commentid])
+    is_my_comment = comment.present? && (comment.comment_by == current_user.id)
```

Comment:
can we extract this boolean into it's own method and add a `?` to the end of `is_my_comment` to specify it's a boolean value?

Reply:
yeah, that sounds like a good idea